lib/src/sql/statements/break.rs

```diff
@@ -0,0 +1,56 @@
+use crate::ctx::Context;
+use crate::dbs::{Options, Transaction};
+use crate::doc::CursorDoc;
+use crate::err::Error;
+use crate::sql::error::IResult;
+use crate::sql::value::Value;
+use derive::Store;
+use nom::bytes::complete::tag_no_case;
+use revision::revisioned;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize, Store, Hash)]
+#[revisioned(revision = 1)]
+pub struct BreakStatement;
+
+impl BreakStatement {
+	/// Process this type returning a computed simple Value
+	pub(crate) async fn compute(
+		&self,
+		_ctx: &Context<'_>,
+		_opt: &Options,
+		_txn: &Transaction,
+		_doc: Option<&CursorDoc<'_>>,
+	) -> Result<Value, Error> {
+		Err(Error::FeatureNotYetImplemented {
+			feature: "BREAK statements",
+		})
+	}
+}
+
+impl fmt::Display for BreakStatement {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		f.write_str("BREAK")
+	}
+}
+
+pub fn r#break(i: &str) -> IResult<&str, BreakStatement> {
```

Comment:
I was not familiar with this convention and don't know how I feel about it 😂

Reply:
`r#something` are raw identifiers, they de-mangle and resolve to the `something` part. See https://doc.rust-lang.org/rust-by-example/compatibility/raw_identifiers.html.